data/wanderers.txt

```diff
@@ -342,14 +342,254 @@ phrase "friendly wanderer"
 		"wrecked "
 		"abandoned "
 	word
-		"worlds."
-		"planets."
-		"places."
-		"lands."
-		"ecosystems."
-		"biospheres."
+		"worlds"
+		"planets"
+		"places"
+		"lands"
+		"ecosystems"
+		"biospheres"
+	word
+		"."
 
+phrase "friendly wanderer"
+	word
+		"You are "
+	word
+		"a welcome "
+		"an honored "
+	word
+		"[visitor, traveller] "
+		"guest "
+	word
+		""
+		"to us"
+		"to our people"
+		"to the Wanderers"
+	word
+		", "
+		" - "
+	word
+		"so long as "
+		"as long as "
+		"[providing, given] "
+	word
+		"you respect our ways"
+		"you respect our worlds"
+		"you respect our ecosystems"
+		"you show respect for our ways"
+		"you show respect for our worlds"
+		"you show respect for our ecosystems"
+		"you live sustainably"
+		"you [consume, devour] no more than you need"
+		"you cause no harm"
+		"you come in peace"
+	word
+		"."
 
+phrase "friendly wanderer"
+	word
+		"We mean "
+		"We [intend, plan] "
+		"We come "
+	word
+		"to "
+		"only to "
+		"[simply, solely] to "
+	word
```

Comment:
I usually try to limit the things in brackets to only a few per sentence, and to avoid having multiple ones back to back, just because it makes the sentence so much harder to read. I'd suggest breaking the bracket phrases up into two separate alternative lines, in these first two "words."